recirq/quantum_chess/quantum_board_test.py

```diff
@@ -1211,10 +1211,10 @@ def test_caching_accumulations_different_repetition_not_cached(board):
         )
     )
     probs1 = b.get_probability_distribution(1)
-    probs2 = b.get_probability_distribution(100)
+    probs2 = b.get_probability_distribution(1001)
```

Comment:
Optional:  it might make sense to change this into a constant so that it is clear that the number of reps is important.

For instance, CACHE_INVALIDATION_REPS maybe?  Then add a comment where you define the constant as to what this is.